docs/TestCoverage.md

```diff
@@ -2533,6 +2533,28 @@ expect(node, inputs=[data, indices.astype(np.int64)], outputs=[y],
 </details>
 
 
+### GatherND
+There are 1 test cases, listed as following:
+<details>
+<summary>gathernd</summary>
+
+```python
+node = onnx.helper.make_node(
+    'GatherND',
+    inputs=['data', 'indices'],
+    outputs=['output'],
+)
+
+data = np.array([[[0, 1], [2, 3]], [[4, 5], [6, 7]]], dtype=np.float32)
+indices = np.array([[[0, 1]], [[1, 0]]], dtype=np.int64)
+output = np.array([[[2, 3]], [[4, 5]]], dtype=np.float32)
```

Comment:
Could you compute output using a reference implementation? The actual value can be kept in a comment such as
```
output = your_implementation(data, index) # output = [[2, 3], [4, 5]]

Reply:
I don't understand why this is needed, sorry. To implement this in Python might not be needed in the context of this file. We can add equations - I feel that is good enough. This file is mostly to generate the node test case that will be tested by a backend - not for humadn understanding....

Reply:
Yes, reference implementation is a requirement for new operators per Op WG's conclusion. :)

Reply:
Added reference implementation as suggested.